src/ngraph/frontend/onnx_import/ops_bridge.hpp

```diff
@@ -16,17 +16,19 @@
 
 #pragma once
 
-#include "core/node.hpp"
-#include "ngraph/node_vector.hpp"
+#include <string>
+
+#include "core/operator_set.hpp"
 
 namespace ngraph
 {
     namespace onnx_import
     {
         namespace ops_bridge
         {
-            NodeVector make_ng_nodes(const onnx_import::Node&);
-        }
+            const OperatorSet& get_operator_set(std::int64_t version);
```

Comment:
Please add `<cstdint>` header.

Reply:
It is there, again what version of changes are you looking at?